client/web/src/enterprise/batches/detail/changesets/ChangesetNode.tsx

```diff
@@ -16,6 +16,9 @@ export interface ChangesetNodeProps extends ThemeProps {
     viewerCanAdminister: boolean
     history: H.History
     location: H.Location
+    enableSelect?: boolean
+    onSelect?: (id: string, selected: boolean) => void
+    isSelected: (id: string) => boolean
```

Comment:
should this one be optional, too?

```suggestion
    isSelected?: (id: string) => boolean
```